dev-games/libnw/files/libnw-1.30.02-C23.patch

```diff
@@ -0,0 +1,210 @@
+https://bugs.gentoo.org/931876
+Fix incompatible pointer cast by pointing to hopefully correct struct
+member.
+Ordering  of {.y,.c} is important so makefile doesn't try and fail to
+regenerate autogenerated file we just edited.
+--- a/libnw/mdly.y
```

Comment:
Gnarly. For such cases, consider seeing if byacc works in future, but good enough for now.